src/navigation/vertical-navigation.html

```diff
@@ -30,7 +30,7 @@
                     'force-hide-secondary-nav-pf': $ctrl.forceHidden,
                     'show-mobile-nav': $ctrl.showMobileNav}">
       <ul class="list-group">
-        <li ng-repeat="item in $ctrl.items" class="list-group-item"
+        <li ng-repeat="item in $ctrl.items | navItems:$ctrl.inMobileState" class="list-group-item"
```

Comment:
Might be easer just to add:

```
'visible-xs': item.mobileOnly
```

to the `ng-class` rather than filtering.

Reply:
but is that acceptable performance wise?  Seems a waste to pull in/construct all that dom/html only to not display it.  Seems like it would be faster just filtering items.